data/campaigns/Northern_Rebirth/scenarios/12a_Get_the_Gold.cfg

```diff
@@ -23,74 +23,82 @@
 
         [part]
-            story= _ "At Hidel’s call Tallin approached and knelt beside Hidel, bowing his head in respect."
+            story= _ "At Hidel’s call Tallin approached. He knelt beside, bowing his head in respect."
         [/part]
 
         {STORY_PART_TALLIN (_ "At your service, sir.")}
 
         {STORY_PART_HIDEL (_ "Eryssa has been... my charge ever... since she was born... I now return... to the earth... from which... I sprang... Please take care... of her.")}
 
-        {STORY_PART_TALLIN (_ "Rest assured, sir, I will place myself between Eryssa and any ill that strength or love may counter.")}
+        {STORY_PART_TALLIN (_ "I will.")}
 
         {STORY_PART_HIDEL (_ "Thank you... Tallin... Now I may rest... in... peace... May you... be victorious...")}
 
         [part]
-            story= _ "With these words Hidel closed his eyes and breathed his last. As Eryssa wept with Hidel’s head in her lap, Tallin approached her, hesitating, and then wrapped her gently in his arms."
+            story= _ "Then Hidel closed his eyes and took one last breath."
+        [/part]
+
+        {STORY_PART_ERYSSA (_ "He's dead!")}
+
+        [part]
+            story= _ "Slowly Tallin aproached her. He hesitated before wrapping her gently in his arms."
             music=love_theme.ogg
         [/part]
         [part]
-            story= _ "Sobbing against Tallin’s chest, Eryssa told him of the many ways Hidel had cherished and warded her since the hour of her birth. How he had comforted her childhood hurts; how in later years, he had guided her in the green paths of the forest, and taught her the deep mysteries of elven-kind."
+            story= _ "Sobbing against Tallin’s chest, Eryssa told him of the many ways Hidel had cherished and warded her and her sister since the hour of their birth. How he had comforted their childhood hurts; how in later years, he had guided them in the green paths of the forest, and taught them the deep mysteries of elven-kind."
         [/part]
         [part]
-            story= _ "A torrent of childhood memories and the depth of her grief left her unable to speak. She and Tallin sat together silently holding each-other for some time."
+            story= _ "The torrent of childhood memories and the depth of her grief left her unable to speak. She and Tallin sat together silently holding each-other for some time."
         [/part]
         [part]
-            story= _ "As Tallin comforted Eryssa, the rest of the party spread out and rallied the scattered elves, and tending to the wounded. This being done, they once again came before Tallin and Eryssa."
+            story= _ "As Tallin comforted Eryssa, the humans dug a large pit where the elves set all the comraids that had died. The dwarves gathered stones and raised a bier over Hidel, burrying him with the arms of all the foes he had slain. Then they came before Tallin and Eryssa."
```

Comment:
Please rewrite this line, I don't think we'll have elves burying dead as a custom but I'm not sure what to do with them yet.

Reply:
I understand elves wouldn't usually' bury, but I also don't think they'd burn them so, I decided if it was burring, burning, or letting crows eat them, burying would be a valid option.

I wanted it to be a scene of tragedy where the races worked together, so...

Reply:
Yes but in this way we're adding lore when there's none elsewhere in mainline. If there were precedence we could pass this off as a one off, but since there is no precedence at all adding this makes it standard canon which I think we need to think about more.

Reply:
Ok. We can leave this open, I'm really not part of the dev group, so I think my suggestions speak for myself, and it'd be up to you.

(I think elves should either send them away on ships or do something with trees, but not hand dead bodies in trees.)

Dwarves, clearly would be buried in stone.

Humans just buried.

Reply:
Well that's why I'm saying this line needs rewriting. You want a tragic scene which is fine, but unfortunately we don't have canon established for elves. Dwarves and humans I think are pretty clear, those make sense to me. Since we're talking about elves here, until we figure out what they do with their dead, we should leave it vague imo.

Reply:
The other question is, does it matter? Would the elves have time to do {elvish burying practice here}? Or would they just try to have crows not eat them and then go kill orcs?

Do you mean I should have the humans/dwarves/elves  "lay them to rest?"  I don't know, I think there needs to be a bit more focus on the respect that they pay Hidel.

Reply:
It matters because people read mainline canon for ideas on what races do or don't do so that affects all of UMC. So we do exactly what you said. Leave the exact wording vague enough while focusing on the respect for Hidel. We don't say they "dug a pit and buried" but "held a grave memorial for their fallen hero" or something like that.

Reply:
The part of "does it matter" I meant that the elves might not be in a position to perform whatever burial rites they do (Which take longer) We could add a comment or reword it to make people know that it isn't what elves normally do. 

I can reword it, but can I open an issue to get back to this after you all think about it?

Reply:
So I suggest rewording it to the vague wording and open an issue to get back to me later. Burial rites is something relevant that we can add into race descriptions at some point so let's brainstorm later.

Reply:
Got it.  I'll do it right now.